ui/run.sh

```diff
@@ -130,6 +132,8 @@ case "$(basename ${0})" in
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy deploy || exit $?
         run ecsdeploy start || exit $?
+	/home/admin/ECS-CommunityEdition/patches/3.6.0.0/vnest_patch
+	o "Please wait for 30 minutes before running next step for services to bootstrap"
```

Comment:
Why?  Use correct ping tool.  `ecs-config ping` is the correct tool.  If it does not work, update the code so that it does work.